script/Ciel/Day 8/08_01_CIEL08_4.txt

```diff
@@ -550,7 +600,8 @@ It seems that she doesn't want to say her name, to say Ciel.
 -- Page 378, Offset 19199.
 -- わたし、志貴のコトはなんだってイイんだけどアイツだけはダメ。貴方の気持ちがアイツに向くだけで許せない」
 // Literally 気持ちが向く feelings turn towards her
-I don't mind you, Shiki, but not her. I won't allow you to be drawn to her."
+I don't mind what you get up to Shiki, but only if it doesn't involve her.
+ I won't let you fall under her sway."
```

Comment:
The second sentence is more like
'You starting to feel for her is all it's gonna take for me to hate you.'
isn't it?